src/script/entity/message/MemberMessage.ts

```diff
@@ -115,7 +118,7 @@ export class MemberMessage extends SystemMessage {
 
     this.showNamedCreation = ko.pureComputed(() => this.isConversationCreate() && this.name().length > 0);
 
-    this.otherUser = ko.pureComputed(() => (this.hasUsers() ? this.userEntities()[0] : new User('', null)));
+    this.otherUser = ko.pureComputed(() => (this.hasUsers() ? this.userEntities()[0] : new User(undefined, null)));
```

Comment:
The ID turned from `''` into `undefined`. We have to observe if this causes complications in the future...